ports/mimxrt10xx/common-hal/busio/UART.c

```diff
@@ -117,7 +117,7 @@ void common_hal_busio_uart_construct(busio_uart_obj_t *self,
     self->timeout_ms = timeout * 1000;
 
     if (self->character_bits != 7 && self->character_bits != 8) {
```

Comment:
Is this if still required?

Reply:
Thanks, I thought I had caught all of these.